docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -553,8 +557,92 @@ as follows:
 	# Frozen normal distribution for MD, representing uncertainty in our effect size
 	bw_md_distribution = norm(mean, std)
 
-Determining Whether A Simulant is Affected
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Determining How A Simulant is Affected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Hemoglobin Level**
+
+For the purposes of our simulation, we assumed that simulants only received an 
+effect from iron fortification if they directly consumed iron fortified foods 
+and that they received no effect of iron fortification on their hemoglobin 
+levels from maternal consumption of iron fortified foods.
+
+Additionally, we assumed that simulants in  begin to eat staple foods as a 
+supplement to breast milk consumption at the age of six months and that the 
+quantity of staple foods consumed as a proportion of total consumption 
+increases linearly from six months to two years of age, at which point it 
+reaches its peak and then remains constant.
+
+Further, we assumed that the full effect of the iron fortification 
+intervention takes six months to achieve and that the effect scales up in a 
+linear fashion between the onset of exposure and six months post exposure (
+this is likely a conservative assumption and the shape likely follows more of 
+a logarithmic curve). Additionally, we assumed that all individuals covered by 
+baseline coverage of iron fortification have been covered for at least six 
+months and therefore have already achieved the full effect of the intervention.
+
+.. todo::
+
+	Add justification and references to support for these assumptions.
+
+Therefore, the effect size of iron fortification on a simulant's hemoglobin 
+level **in the baseline scenario** should be determined as follows:
+
+.. code-block:: Python
+	
+	# at the draw level
+	MD = hb_md_distribution.rvs()
```

Comment:
hb must stand for hemoglobin.  what does MD stand for?  add as a comment

Reply:
md here is mean difference. I will add this! Thanks